Northstar.Client/mod.json

```diff
@@ -98,6 +98,17 @@
 		{
 			"Path": "ui/controller_prompts.nut",
 			"RunOn": "UI"
+		},
+		{
+			"Path": "ui/ui_server_dialog.nut",
+			"RunOn": "UI"
+		},
+		{
+			"Path": "cl_server_dialog.gnut",
+			"RunOn": "CLIENT",
+			"ClientCallback": {
+				"After": "serverDialogueOnClientInit"
```

Comment:
Why is the S lowercase :(

Reply:
idk, I can fix it if you want

Reply:
fixed on this commit [22dd78d](https://github.com/R2Northstar/NorthstarMods/pull/380/commits/22dd78d4eb588058b479df5f06ac3a4c026013fb)